client/src/components/Navbar/components/Drawer/index.tsx

```diff
@@ -0,0 +1,234 @@
+import {
+  Button,
+  Classes,
+  Drawer as BPDrawer,
+  H3,
+  Icon,
+  InputGroup,
+  FormGroup,
+  Spinner
+} from "@blueprintjs/core";
+import * as React from "react";
+import { Mutation } from "react-apollo";
+import { useSelector } from "react-redux";
+import { useApolloClient } from "react-apollo-hooks";
+
+import "./style.less";
+import StringSelect from "../../../selects/stringSelect";
+import { IReduxStore } from "../../../../types";
+import sourceSchemas from "../../../../services/selectedNode/sourceSchemas/reducer";
+
+const credential = require("../../../../graphql/queries/credential.graphql");
+const upsertCredential = require("../../../../graphql/mutations/upsertCredential.graphql");
```

Comment:
nit: I tend to use absolute paths in my imports as it improves readability. eg: `import StringSelect from 'components/selects/stringSelect';` I think it shuyold work out-of-the-box with `react-scripts`

Reply:
Great, I didn't know one could do this! Fixed in 8f3a6c1.
I'll slowly refactor other components and start using absolute paths instead.